zuul.d/jobs.yaml

```diff
@@ -204,6 +204,7 @@
         TEST_IRONIC_DEPLOY_INITRD: 'http://10.245.161.162/swift/v1/images/ironic-python-agent.initramfs'
         TEST_IRONIC_DEPLOY_VMLINUZ: 'http://10.245.161.162/swift/v1/images/ironic-python-agent.kernel'
         TEST_IRONIC_RAW_BM_IMAGE: 'http://10.245.161.162/swift/v1/images/baremetal-ubuntu-focal.img'
+         TEST_TRILIO_LICENSE: /tmp/tvault-license.lic
```

Comment:
indentation is off by one space - not sure if that can lead to a yaml parsing error?